src/eventDispatchers/mouseEventHandlers/addNewMeasurement.js

```diff
@@ -1,9 +1,15 @@
 import external from '../../externalModules.js';
 import { addToolState } from '../../stateManagement/toolState.js';
 import { moveHandle, moveNewHandle } from '../../manipulators/index.js';
+import debug from 'debug';
+
+const log = debug('cornerstoneTools')
```

Comment:
I'm not super familiar with this library, but it looks like we're create a debugger in multiple locations. Should we be creating a single `logger` instance in the store, and then extending it, as appropriate, in each module?

Reply:
I've done something like this now. I'm not worried about creating too many instances of the logger though. That's not likely to cause performance issues unless someone starts doing that inside a loop or event handler.